setup.cfg

```diff
@@ -44,3 +44,4 @@ markers =
 show-source = True
 max-line-length = 80
```

Comment:
I think we should also set this to `88`: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length